### PR TITLE
Release ruby 3.2.3

### PR DESCRIPTION
**Pull request by nagachika**

### CODE REVIEW

File: ja/news/_posts/2024-01-18-ruby-3-2-3-released.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+author: "nagachika"
+translator:
+date: 2024-01-18 09:00:00 +0000
+lang: ja
+---
+
+Ruby 3.2.3 がリリースされました。
+
+このリリースでは多くの不具合修正を行なっています。
+詳しくは [GitHub releases](https://github.com/ruby/ruby/releases/tag/v3_2_2) を参照してください。
```
[the remark this answers is not in the document]
Comment (nagachika) [reply ja/news/_posts/2024-01-18-ruby-3-2-3-released.md:13]: thanks!!

File: en/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md [hunk 2]
```diff
@@ -24,7 +24,8 @@ The `uri` gem version 0.12.1 and all versions prior 0.12.1 are vulnerable for th
 We recommend to update the `uri` gem to 0.12.2. In order to ensure compatibility with bundled version in older Ruby series, you may update as follows instead:
 
 * For Ruby 3.0: Update to `uri` 0.10.3
-* For Ruby 3.1 and 3.2: Update to `uri` 0.12.2
+* For Ruby 3.1: Update to `uri` 0.12.2
+* For Ruby 3.2: Update to `uri` 0.12.2, or update to ruby-3.2.3.
```
Comment (sorah) [inline en/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md:28]:
```suggestion
* For Ruby 3.2: Update to `uri` 0.12.2, or update to Ruby 3.2.3
```

File: en/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md [hunk 3]
```diff
@@ -40,4 +41,5 @@ Thanks to [nobu](https://github.com/nobu) for fixing this issue.
 
 ## History
 
+* Append recommended action for Ruby 3.2 at 2024-01-18 12:00:00 (UTC)
```
Comment (sorah) [inline en/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md:44]:
```suggestion
* Added a new recommended action for Ruby 3.2 at 2024-01-18 12:00:00 (UTC)
```

File: ja/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md [hunk 4]
```diff
@@ -40,4 +41,5 @@ ReDoS 脆弱性のセキュリティ修正を含む、uri gem 0.12.2、0.10.3 
 
 ## 更新履歴
 
+* 2024-01-18 19:00:00 (JST) ruby 3.2 での対応について追記
```
Comment (sorah) [inline ja/news/_posts/2023-06-29-redos-in-uri-CVE-2023-36617.md:44]:
```suggestion
* 2024-01-18 19:00:00 (JST) Ruby 3.2 での対応を追加
```

**Comment by stanhu** [comment]
@nagachika Thanks for this! Are there plans for a 3.1.5 release? I'd like to get the fix for https://github.com/ruby/ruby/pull/8246.

**Comment by hsbt** [comment]
@stanhu We have plans for that, so please wait.

**Pull request merged**